### DISCUSSION TITLE
Sending a Raw Query Body

### DISCUSSION THREAD

**Discussion by GioValentin** [Jeroen-G/Explorer#109] [Q&A]
How could I send a raw query ? 

Model::search(raw_query_formatted_in_JSON)

**Comment by Jeroen-G** [comment]
You can't :)

**Reply by Jeroen-G** [reply]
What you can _try_ is creating a custom query syntax object, or using the ES adapter directly.

**Reply by Jeroen-G** [reply]
But that is more advanced stuff that you need to be comfortable with.

**Reply by GioValentin** [reply]
I was able to get it resolved by extending the the Explorer service provider with some custom logic.

**Reply by Jeroen-G** [reply]
Do you have the code online? I'm curious :)
